package com.dragonminez.common.init.item.weapons.model;

import com.dragonminez.Reference;
import com.dragonminez.common.init.item.weapons.YajirobeKatanaItem;
import net.minecraft.resources.ResourceLocation;
import software.bernie.geckolib.model.GeoModel;

public class YajirobeKatanaModel extends GeoModel<YajirobeKatanaItem> {
    @Override
    public ResourceLocation getModelResource(YajirobeKatanaItem animatable) {
        return new ResourceLocation(Reference.MOD_ID, "geo/weapons/yajirobe_katana.geo.json");
    }

    @Override
    public ResourceLocation getTextureResource(YajirobeKatanaItem animatable) {
        return new ResourceLocation(Reference.MOD_ID, "textures/item/weapons/yajirobe_katana.png");
    }

    @Override
    public ResourceLocation getAnimationResource(YajirobeKatanaItem animatable) {
        return null;
    }
}
